### DISCUSSION TITLE
Update Pocketbase in production with or without update command

### DISCUSSION THREAD

**Discussion by einarpersson** [pocketbase/pocketbase#3186] [Q&A] [answered]
Hi!

I am about to update my production db from Pocketbase 0.16.x to 0.17.y. In the [release log](https://github.com/pocketbase/pocketbase/releases/tag/v0.17.0) for 0.17.0 I can read _"Make sure to have a backup of your pb_data before updating if you are not using the update command with the prebuilt executable."_ 

I am hosting my app on fly.io and deploy from a Dockerfile. When I update I don't use this command, I just edit the Pocketbase version in the Dockerfile. The pb_data is a fly.io volume which has snapshots / backups.

**Question:** Does the update command do anything else than just update the binary and create a backup (optionally)? Does it alter anything in `pb_data`? Because, in that case I suppose I have to ssh into the machine and run that command when updating?

Or, can I keep my current update strategy? (as long as the release log for any future update does not explicitly provides a mandatory update strategy)

```Dockerfile
FROM alpine:latest

RUN apk add --no-cache \
  unzip \
  # this is needed only if you want to use scp to copy later your pb_data locally
  openssh

ARG PB_VERSION=0.16.3

# download and unzip PocketBase
ADD https://github.com/pocketbase/pocketbase/releases/download/v${PB_VERSION}/pocketbase_${PB_VERSION}_linux_amd64.zip /tmp/pb.zip
RUN unzip /tmp/pb.zip -d /pb/

EXPOSE 8080

COPY "/pb_migrations" "/pb/pb_migrations" 

# start PocketBase
CMD ["/pb/pocketbase", "serve", "--http=0.0.0.0:8080"]
```

**Comment by ganigeorgiev** [comment, marked as answer]
Bumping just the version in your Dockerfile or fly.toml is fine.

The `update` command only creates a backup and replaces the old binary with the new one (_aka. more or less the same as what you currently do with changing the Dockerfile version_).

If there is any need for altering `pb_data` this usually happens via system migration, but that is executed automatically on the first `serve` of the new executable.
